examples/pymc3_howto/lasso_block_update.ipynb

```diff
@@ -4,40 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>The <code>notebook_id</code>  is the key we'll use from other notebooks (or versioned docs) to link here, it should be somehting notebook specific like the filename. I will update the advise to say it must be the filename, you are at least the 4th person confused by that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>